tardis/montecarlo/montecarlo_numba/single_packet_loop.py

```diff
@@ -47,6 +49,7 @@ def single_packet_loop(
     numba_plasma : tardis.montecarlo.montecarlo_numba.numba_interface.NumbaPlasma
     estimators : tardis.montecarlo.montecarlo_numba.numba_interface.Estimators
     vpacket_collection : tardis.montecarlo.montecarlo_numba.numba_interface.VPacketCollection
+    tracked_rpackets_properties: tardis.montecarlo.montecarlo_numba.numba_interface.RPacketCollection
```

Comment:
This is not the variable name in the argument

Reply:
Resolved